### ISSUE TITLE
When animate nodes names are translated, the Chinese will not display properly

### ISSUE THREAD

**Issue by cx1132692358** [JacquesLucke/animation_nodes#1070]
Obviously he only supports English, but what can I do to make it show Chinese (I tried Unicode)

**Comment by cx1132692358** [comment]
node name

**Comment by cx1132692358** [comment]
def setup(self):
        self.label = "B网格网格BMesh Mesh"
这个方法适用于所有的ANIMATION NODES的多语言化 期待能有所帮助

**Comment by OmarEmaraDev** [comment]
I am not sure what you are asking here. Can you elaborate?

**Comment by cx1132692358** [comment]
Hello, sir.
Thank you for your reply.
I am a blender user in China. With my understanding of 3D, I planned to translate animation nodes in March.
It is obvious that I encountered a problem if I replaced the string in English directly. 
Fortunately, I found a solution. That is to say, when a new node is built, a bilingual label is given at the same time. It's like the picture below. At the same time, I marked it all in the code with # cn.
Of course, I also encountered some situations that could not be added to Chinese, so I changed the content of the annotation.
In short, after a month of hard work, I finished it.
Although I followed the spirit of open source and opened it to everyone for free, I gradually lost the confidence to defend it because of illness and the reselling of businessmen. So this is my last version（2019/04/13）, which contains Chinese and English. I hope it will be useful to you.
Then I started another job to help more people understand blender 2.8.
After a year of hard work, I have completed more than 300 sets of blender 2.8 tutorials, which are completely free of charge.
This is my channel:https://space.bilibili.com/35723238
And my main open courses:https://www.bilibili.com/video/av55550709
Finally, I have a question.
In the future, I want to create an open tutorial on animation nodes to help Chinese learners. I choose to continue to wait for blender updates or to use animation nodes plug-ins.
Finally, sir
Although we have never met, if you need my help, please let me know.
Your fans

只剩一瓶辣椒酱









------------------ 原始邮件 ------------------
发件人:&nbsp;"Omar Emara"<notifications@github.com&gt;;
发送时间:&nbsp;2019年10月7日(星期一) 凌晨2:25
收件人:&nbsp;"JacquesLucke/animation_nodes"<animation_nodes@noreply.github.com&gt;;
抄送:&nbsp;"1132692358"<1132692358@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [JacquesLucke/animation_nodes] When animate nodes names are translated, the Chinese will not display properly (#1070)




I am not sure what you are asking here. Can you elaborate?

**Comment by cx1132692358** [comment]
Hello, sir.
Thank you for your reply.
I am a blender user in China. With my understanding of 3D, I planned to translate animation nodes in March.
It is obvious that I encountered a problem if I replaced the string in English directly.
Fortunately, I found a solution. That is to say, when a new node is built, a bilingual label is given at the same time. It's like the picture below. At the same time, I marked it all in the code with # cn.
Of course, I also encountered some situations that could not be added to Chinese, so I changed the content of the annotation.
In short, after a month of hard work, I finished it.

**Comment by cx1132692358** [comment]
![image](https://user-images.githubusercontent.com/18584564/66329880-e1266480-e961-11e9-8b62-3e709d2b162f.png)

**Comment by cx1132692358** [comment]
![image](https://user-images.githubusercontent.com/18584564/66329914-f13e4400-e961-11e9-96cd-e4221cc71ada.png)

**Comment by cx1132692358** [comment]
[animation_nodes.zip](https://github.com/JacquesLucke/animation_nodes/files/3698467/animation_nodes.zip)

**Comment by cx1132692358** [comment]
lthough I followed the spirit of open source and opened it to everyone for free, I gradually lost the confidence to defend it because of illness and the reselling of businessmen. So this is my last version（2019/04/13）, which contains Chinese and English. I hope it will be useful to you.
Then I started another job to help more people understand blender 2.8.
After a year of hard work, I have completed more than 300 sets of blender 2.8 tutorials, which are completely free of charge.
This is my channel:https://space.bilibili.com/35723238
And my main open courses:https://www.bilibili.com/video/av55550709
Finally, I have a question.
In the future, I want to create an open tutorial on animation nodes to help Chinese learners. I choose to continue to wait for blender updates or to use animation nodes plug-ins.
Finally, sir
Although we have never met, if you need my help, please let me know.
Your fans

**Comment by cx1132692358** [comment]
Although I followed the spirit of open source and opened it to everyone for free, I gradually lost the confidence to defend it because of illness and the reselling of businessmen. So this is my last version（2019/04/13）, which contains Chinese and English. I hope it will be useful to you.
Then I started another job to help more people understand blender 2.8.
After a year of hard work, I have completed more than 300 sets of blender 2.8 tutorials, which are completely free of charge.
This is my channel:https://space.bilibili.com/35723238
And my main open courses:https://www.bilibili.com/video/av55550709
Finally, I have a question.
In the future, I want to create an open tutorial on animation nodes to help Chinese learners. I choose to continue to wait for blender updates or to use animation nodes plug-ins.
Finally, sir
Although we have never met, if you need my help, please let me know.
Your fans
只剩一瓶辣椒酱

**Comment by OmarEmaraDev** [comment]
I see, thanks for your interest in translating Animation Nodes. I think there might be better ways to do such translations, but I haven't done something like this before. I will try to look into it at some point and get back to you.

**Comment by cx1132692358** [comment]
All right. Thank you for your work.
